src/stt/src/SteinerTreeBuilder.cpp

```diff
@@ -115,4 +132,34 @@ Tree SteinerTreeBuilder::makeTree(std::vector<int>& x,
   return tree;
 }
 
+int SteinerTreeBuilder::computeHPWL(odb::dbNet* net)
```

Comment:
You should verify that the iterms/bterms are actually placed.

Reply:
If some term isn't placed, should I error out or throw a warning message? If throw a warning, is it ok to return 0 as HPWL?

Reply:
I can't think of a case where you want a steiner tree for a net with unplaced instances so let's make it an error.

Reply:
I think it best to just check this upfront in makeSteinerTree rather than only when an hpwl limit is set.  Otherwise turning it on will cause a hidden error to emerge.